modules/bibupload/lib/batchuploader.py

```diff
@@ -105,7 +105,7 @@ def task_run_core():
                     shutil.copy(os.path.join(files_dir, metafile), filename)
                     # Send bibsched task
                     mode = "--" + folder
-                    jobid = str(task_low_level_submission('bibupload', 'batchupload', mode, filename))
+                    jobid = str(task_low_level_submission('bibupload', 'batchupload', mode, filename, '-N', 'daemon'))
```

Comment:
Mmh. Seems like the user was already set to `batchupload`. With this operation we are just setting the bibupload specific name to `bibupload:daemon`. *Per se*, should be OK, since this can help catalogers identifying in the bibsched queue the bibuploads issued by reading the batchupload directories.